frog-utils/src/index.js

```diff
@@ -131,3 +135,87 @@ export const wordWrap = (text: string, maxLength: number): string[] => {
   }
   return result;
 };
+
+const groupchars = 'ABCDEFGHIJKLMNOPQRSTUWXYZ123456789'.split('');
+export const getSlug = (n: number) =>
+  shuffle(groupchars)
+    .slice(0, n)
+    .join('');
+
+type ReactivePropsT = {
+  path: string | string[],
+  dataFn: Object,
+  type: 'textarea' | 'textinput'
+};
+
+export class ReactiveText extends Component {
```

Comment:
Would it be nicer to move this Component to `frog-utils/ActivityComponents`. Along with the type definition